app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+  belongs_to :user, foreign_key: true
+  after_save :update_posts_counter
+
+  def check_last_five_posts
+    comments.last(5)
```

Comment:
I learnt a new method from you today!

Reply:
Thanks, however, I was wrong because the `.last(5)` will give me the comments in ascending order (for example 1, 2, 3, 4, 5) so I just did my research and then replace the method with a better one which returns the comments from the most recent to the oldest comment (with the 5 range, of course, it would be like 5, 4, 3, 2, 1). Thank you so much for all your feedback 👍🏼 .

Reply:
Interesting @angeluray. Let me recheck this. Good job though in noticing this and mentioning it as well.